app/models/activity.rb

```diff
@@ -160,7 +160,7 @@ def actual_total_for_report_financial_quarter(report:)
   end
 
   def forecasted_total_for_report_financial_quarter(report:)
-    @forecasted_total_for_report_financial_quarter ||= planned_disbursements.where(period_start_date: report.created_at.all_quarter).sum(:value)
+    @forecasted_total_for_report_financial_quarter ||= forecasted_total_for_date_range(range: report.created_at.all_quarter)
```

Comment:
I might be confused by this, but given reports `has_many :planned_disbursements`, couldn't you do:  `report.planned_disbursements.sum(:value)` ?

Reply:
No you're not confused, I think we originally hung the query off the activity, because originally I was looking for PDs that were for a date range *and* a report, but now we're just looking for ones in a date range. I'll see if I can make it better/more obvious

Reply:
Sorry ignore my other comment. They need to be constrained to a date range because of https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/pull/579#discussion_r472843701